manim/mobject/mobject.py

```diff
@@ -1902,6 +1905,7 @@ def set_colors_by_radial_gradient(
         )
         return self
 
+    @internal
```

Comment:
why?

Reply:
There was already a `set_colors_by_gradient` that used this internally and returning `Self`, so I figured that should be the public one.